mars_craters_starting_kit.ipynb

```diff
@@ -130,21 +134,19 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "#### Installation of libraries\n",
+    "#### Installation of libraries and `ramp-workflow`\n",
     "\n",
-    "To get this notebook running and test your models locally using the `ramp_test_submission`, we recommend that you use the Python distribution from [Anaconda](https://www.anaconda.com/download/) or [Miniconda](https://docs.anaconda.com/docs_oss/conda/install/quick#miniconda-quick-install-requirements). "
+    "To get this notebook running and test your models locally using the `ramp_test_submission`, we recommend that you use the Python distribution from [Anaconda](https://www.anaconda.com/download/) or [Miniconda](https://conda.io/miniconda.html). "
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {
-    "collapsed": true
-   },
+   "metadata": {},
    "outputs": [],
    "source": [
-    "# !conda install -y -c conda conda-env     # First install conda-env to ease the creation of virtual envs in conda\n",
-    "# !conda env create                        # Uses the local environment.yml file to create the 'mars-craters' env"
+    "# !conda env create -f environment.yml   # use the environment.yml file to create the 'mars-craters' env\n",
+    "# !source activate mars-craters          # activate the virtual environment"
```

Comment:
If you have the latest `conda`, the above will be `conda activate` instead of `source activate` (but not sure if `source` activate keeps working)